apps/pm-user/src/components/molecules/quickInfoBar/quickInfoBar.module.scss

```diff
@@ -0,0 +1,24 @@
+.quickInfoBar {
+  & p{
+    font-weight: 600;
+    color: #5c5fee;
+  }
+  :global {
+    .ant-segmented {
+      background: transparent;
+    }
+    .ant-segmented-item-label {
+      padding: 5px;
```

Comment:
Using multiples of 4 is better.

Reply:
can elaborate it?

Reply:
Chaitanya suggested that it is a good practice to use multiples of 4 for all CSS properties.

Reply:
ok I dont know it. you can add in it next PR